distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -85,14 +85,20 @@ To set multi-valued attributes, use a separate override for each value. For exam
 `*topic.keyword=PPI*` +
 `*topic.keyword=radiology*`
 
-Attributes will only be overridden if they are part of the <<_metacard_type, metacard type>>, however the attributes do not have to be present on the metacard to be set.
+Attributes will only be overridden if they are part of the <<_metacard_type, metacard type>> or are <<_attribute_injection_definition,injected>>, however the attributes do not have to be present on the metacard to be set.
+
+Common attributes that can be overridden on all metacards can be found in the <<_catalog_taxonomy, catalog taxonomy tables>> labeled with *Injected by default*.
```

Comment:
✏️  Some places have a period in `*Injected by default.*` this while others don't. These can probably be consistent.